nodes/docker-images.md

```diff
@@ -160,21 +161,21 @@ An example init command will look similar to below:
 
 ```bash-vue [Mainnet Beta]
 docker run -e NODE_TYPE=$NODE_TYPE -e P2P_NETWORK=$NETWORK \
```

Comment:
This should be added here:
```
-e CELESTIA_HOME=/celestia
```

or a section above should be added exporting a variable that can be used here like the others.

Or the other option would be to modify the default docker environment variable to use the `/celestia` path instead of `/home/celestia`.

Or you may want to switch the other commands back to using `/home/celestia` for the mounted volume instead of `/celestia` to maintain persistence.

Reply:
i tested the full command when i edited this doc and it worked as expected. `-v $HOME/my-node-store:/celestia \` does the trick

Reply:
what OS are you on? i think that may be the issue. this doc works for mac and linux.

Reply:
I've executed it both within windows and Ubuntu WSL. Are you sure it is actually persisting in your testing? The command executes successfully as written for me as well, it just doesn't actually persist anything. So after running it you can check in the `$HOME/my-node-store` folder on your host machine and find no files or directories were created (of course you should wipe the directory first to ensure it is empty to begin with and isn't using the data from previous runs).

So while I think all of the commands in the docs will not currently throw any errors, I do believe they are not correct as currently written as they will not achieve the intended goal of a persistent node. Perhaps I messed up somehow though.

Reply:
got it, i’ll test to verify!

Reply:
The original comment in this thread was to fix the persistence issue I had encountered.